extends RichTextLabel
var slapoha = false

func _physics_process(delta: float) -> void:
	
	if slapoha == false:
		if Global.chat_on == true:
			$"../../Timer".start()
			slapoha = true
			print("FOI")

func chat():
	$".".text += (Users.pick_random()) + ": "+ mensagens.pick_random() + "\n"
	


var mensagens = [
	"Oieee!", "A Live jÃ¡ ComeÃ§ou?", "Caramba!!!", "To curtindo a live!", "Pog! :O",
	"Gente a live jÃ¡ comeÃ§ou?", "Jogo Novo?", "Quando VocÃª vai jogar Crossout???",
	"chegueiii", "salveee famÃ­lia", "primeirooo!", "boa noite tropaaa",
	"vamo que vamo ðŸ”¥", "jÃ¡ cheguei dando like", "opa opa opa", "cheguei cedo hj",
	"fui notificado agora", "boraaaaa", "finalmenteee kkk", "salve streamer brabo",
	"partiu gameplay", "a tropa chegou ðŸ˜Ž", "quero ver jogando no hard", "abriu a live cedo hoje?",
	"me nota pls", "tÃ´ desde o comeÃ§o!", "qual jogo vai ser hoje?", "bora bater recorde",
	"cheguei atrasado?", "eitaaa jÃ¡ comeÃ§ou", "qual jogo hj?", "vamo pra cima",
	"essa live promete", "cheguei pra dar sorte", "jÃ¡ peguei minha pipoca ðŸ¿", "streamer mais brabo do BR",
	"bora clipar tudo", "live raiz demais", "jÃ¡ tava com saudade", "fala comigo chat",
	"essa skin Ã© linda", "tÃ¡ rodando liso aÃ­?", "vai zerar hoje?", "chat tÃ¡ on fire",
	"tropa representando", "manda salve pra mim", "hoje vai ser histÃ³rico", "sÃ³ vitÃ³ria"
]
var Users = [
	"XxDragonSlayerxX", "NoobMaster22", "LuluGamer", "TropaDoZap",
	"SniperBR", "Darkzinho", "ClutchGod", "AnaPlay",
	"SpeedRunMano", "Pixelada", "R4t0L0k0", "BiaFPS",
	"ShadowBR", "ReiDoClutch", "TeteuGame", "N1njaDoRole",
	"GatinhoPro", "ZeroPing", "FuriaDoTeclado", "JuJuCraft",
	"BatataQuente", "SniperDeChinelo", "MestreDoLag", "ClaudinhoPRO",
	"SlaMano", "FoguetinhoBR", "RachaTela", "M1nerva",
	"DogaoSemFreio", "ZoeiraTotal", "PixelDoMal", "KibeAssassino",
	"ManoDoX1", "ToSemSono", "LendarioBR", "CafÃ©ComLeite",
	"RatoDeLan", "TioDoPavÃª", "NoiaDoGame", "ProPlayerDeMesa"
]


func _on_timer_timeout() -> void:
	chat()
